frontend/src/components/SepForm.js

```diff
@@ -46,11 +50,16 @@ const SepForm = observer(() => {
   const classes = useStyles()
   const rootStore = useContext(rootStoreContext)
   const participantStore = rootStore.ParticipantStore
+  const [sites, setSites] = useState([])
```

Comment:
I added a local state object for the site list, but I don't think this is the right solution. The Formik form is not technically part of the SepForm component, so the updates to the ParticipantStore object don't appear inside the form. I'm having trouble finding the correct syntax for wrapping the Formik form in observer(). I found this exchange that explains what the issue is, but it does not give me a solution I can directly use. https://github.com/mobxjs/mobx/issues/1505

Reply:
Yes. I think I mentioned this somewhere  back in the issue ( having the site be a mobx piece of state). This should not be an observer connected component at all, actually, if possible. Can you pass down sites array, current site, handle site change method, and form handlers from Sidebar?

Reply:
I have removed the observer wrapper from SepForm in the latest code update, and moved the site related variables to props, but I left the participantStore reference in SepForm as it is used for quite a few other things. It appears to still work okay without the observer wrapper although I don't know if using it this way is considered acceptable.